lib/puppet/reports/datadog_reports.rb

```diff
@@ -135,9 +136,14 @@ def process
     end
 
     facts = Puppet::Node::Facts.indirection.find(host).values
-    dog_tags = REPORT_FACT_TAGS.map { |name| "#{name}:#{facts.dig(*name.split('.'))}" }
-
-    Puppet.debug "Sending events for #{@msg_host} to Datadog"
+    facts_tags = REPORT_FACT_TAGS.map { |name| "#{name}:#{facts.dig(*name.split('.'))}" }
+    trusted_facts = (Puppet.lookup(:trusted_information) { Hash.new }).to_h
+    trusted_facts = { "trusted" => trusted_facts.to_h }
```

Comment:
To make sure `report_trusted_fact_tags` is consistent with `trusted_facts_to_tags`. When accessing the `$trusted` variable from puppet code, do you have to use the `trusted.*` prefix?

Reply:
From the examples section in Puppet docs [1], it looks like the `trusted` prefix doesn't have to be used when accessing `$trusted`, so I'm going to remove it from this PR and merge it.

[1] https://puppet.com/docs/puppet/5.5/lang_facts_and_builtin_vars.html#examples